config/locales/form/en.yml

```diff
@@ -110,8 +105,7 @@ en:
           title: Please provide the latitude and longitude of the area
           popupdescription_html:
             'Please indicate the latitude and longitude of your commitment area. 
-            If unknown, please use this website: <a href="https://www.latlong.net/" target="_blank">https://www.latlong.net/</a>. 
-            If you are using the link to select the centre of your area, this does not need to be exact.'
+            If unknown, please use this website: <a href="https://www.latlong.net/" target="_blank">https://www.latlong.net/</a>.'
```

Comment:
Are the quotes around this necessary?

Reply:
i think so. it doesn't handle the colon in the string properly otherwise, and assumes it is a new key